draft-ietf-quic-transport.md

```diff
@@ -926,6 +926,23 @@ them with other connection IDs for the same connection.  As a trivial example,
 this means the same connection ID MUST NOT be issued more than once on the same
 connection.
 
+Packets with long headers include at least two connection IDs, which are used to
+establish the connection IDs that are used for the connection when established,
+see {{negotiating-connection-ids}} for details.
+
+Packets with short headers ({{short-header}}) only include the Destination
+Connection ID and omit the explicit length.  The length of the Destination
+Connection ID field is expected to be known to endpoints.  Endpoints using a
+connection-ID based load balancer could agree with the load balancer on a fixed
```

Comment:
connection-ID -> Connection ID

Reply:
I think what was meant here is "Connection ID-based load balancer" (dash between ID and based).  This would look odd because Connection ID is two words.  This can be reworded as "using a load balancer based on Connection IDs."